src/main/scala/ai/privado/passes/PropertyParserPass.scala

```diff
@@ -128,19 +128,23 @@ class PropertyParserPass(cpg: Cpg, projectRoot: String, ruleCache: RuleCache, la
 
   private def getDotenvKeyValuePairs(file: String): List[(String, String)] = {
     val envProps = new Properties()
-    Source
-      .fromFile(file)
-      .getLines()
-      .filter(line => line.trim.nonEmpty && !line.startsWith("#"))
-      .foreach(line => {
-        try {
-          val Array(key, value) = line.split("=", 2)
-          envProps.setProperty(key, value)
-        } catch {
-          case e: Throwable =>
-            logger.debug(s"Error splitting the required line. ${e.toString}")
-        }
-      })
+    try {
+      Source
+        .fromFile(file)
+        .getLines()
+        .filter(line => line.trim.nonEmpty && !line.startsWith("#"))
```

Comment:
What happens for multi line comments?

Reply:
How, If there are configs which are commented, nodes will get created for them

Reply:
Ah, this might be a case. But as long as I can remember,  there are no multiline comments in .env files. Let me check again and add support if that's not the case.